s3gw documentation (#212)
epinio/docs: docs/installation/install_epinio.md

Epinio is using an S3 compatible storage to store the application source code.
This chart will install [Minio](https://min.io/) when `.Values.minio.enabled` is
true (default).

> In addition to Minio, Epinio offers [s3gw](https://s3gw.io/) as another internal S3 compatible storage. It can be installed when `.Values.minio.enabled` is `false` and `.Values.s3gw.enabled` is `true`.

Reviewer: ```suggestion
In addition to Minio, Epinio offers [s3gw](https://s3gw.io/) as another internal S3 compatible storage. It is installed when `.Values.minio.enabled` is set to `false` and `.Values.s3gw.enabled` is set to `true`.
```

> Both internal S3 compatible storage can be configured to use a user-defined storageClass. If no StorageClass is defined, the default storageClass is used. The custom storageClass can be set for Minio as `.Values.persistance.storageClass` value or as `.Values.s3gw.storageClass.name` value for s3gw.

Reviewer: ```suggestion
Both choices for internal S3 compatible storage can be configured to use a user-defined storageClass. If no StorageClass is defined, the default storageClass is used. For Minio the custom storageClass is set via the value of `.Values.persistance.storageClass`. For s3gw the  custom storageClass is set via the value of `.Values.s3gw.storageClass.name`.
```
